lib/vsc/utils/script_tools.py

```diff
@@ -36,13 +36,29 @@
 
 from copy import deepcopy
 
+import logging
 from vsc.utils import fancylogger
 from vsc.utils.availability import proceed_on_ha_service
 from vsc.utils.generaloption import SimpleOption
 from vsc.utils.lock import lock_or_bork, release_or_bork, LOCKFILE_DIR, LOCKFILE_FILENAME_TEMPLATE
-from vsc.utils.nagios import SimpleNagios, NAGIOS_CACHE_DIR, NAGIOS_CACHE_FILENAME_TEMPLATE, NAGIOS_EXIT_OK
+from vsc.utils.nagios import (
+    SimpleNagios, NAGIOS_CACHE_DIR, NAGIOS_CACHE_FILENAME_TEMPLATE, NAGIOS_EXIT_OK,
+    exit_from_errorcode
+)
+from vsc.utils.timestamp import (
+    convert_timestamp, write_timestamp, retrieve_timestamp_with_default,
+    convert_to_unix_timestamp)
 from vsc.utils.timestamp_pid_lockfile import TimestampedPidLockfile
 
+DEFAULT_TIMESTAMP = "20140101000000Z"
+TIMESTAMP_FILE_OPTION = 'timestamp_file'
+DEFAULT_CLI_OPTIONS = {
+    'start_timestamp': ("The timestamp form which to start, otherwise use the cached value", None, "store", None),
+    TIMESTAMP_FILE_OPTION: ("Location to cache the start timestamp", None, "store", None),
+}
+MAX_DELTA = 3
+MAX_RTT = 2 * MAX_DELTA + 1
```

Comment:
what is RTT? Regie voor Telegraaf en Telefoon? :wink:

Reply:
Don't ask me, I took it from the Sync class code, it's for determining is some time is out of sync.

Reply:
round trip time